pkg/splunk/enterprise/indexercluster.go

```diff
@@ -123,6 +126,10 @@ func ApplyIndexerCluster(client splcommon.ControllerClient, cr *enterprisev1.Ind
 
 	// no need to requeue if everything is ready
 	if cr.Status.Phase == splcommon.PhaseReady {
```

Comment:
Are we sure that we need the MC only when the reconcile is complete? While scaling up/down?

Reply:
Yes, scaling up/down will be taken here